protobuf/decode.go

```diff
@@ -141,6 +141,8 @@ func (msg *Substate_TxMessage) decode(lookup getCodeFunc) (*substate.Message, er
 	// Berlin hard fork, EIP-2930: Optional access lists
 	var accessList types.AccessList = []types.AccessTuple{}
 	switch txType {
+	case Substate_TxMessage_TXTYPE_SETCODE:
+		panic("setcode tx type is not supported")
```

Comment:
This func returns an error. Instead of panicking for unsupported operation, rather return a new error
```suggestion
		errors.New("setcode tx type is not supported")
```